Don't derive test classes in test_bugs.py from BaseTestCase.

nphilipp opened the pull request:
On the one hand, this still derives from `unittest.TestCase`, which
breaks use of some PyTest features in hard to debug ways. On the other
hand, it also adds unnecessary overhead in form of the test database
that it creates for every called test method.

Signed-off-by: Nils Philippsen <nils@redhat.com>

ryanlerch reviewed (changes requested):
Looks like there are 2 self.asserts that i missed in the initial commit on this file, that still worked because i changed them to basetestcase...

https://github.com/fedora-infra/bodhi/pull/3611/files#diff-5d496e504a611a29cbe4cb4888b45c27R298-R301

nphilipp committed:
Don't derive test classes from BaseTestCase.

On the one hand, this still derives from `unittest.TestCase`, which
breaks use of some PyTest features in hard to debug ways. On the other
hand, it also adds unnecessary overhead in form of the test database
that it creates for every called test method.

Signed-off-by: Nils Philippsen <nils@redhat.com>

mergify[bot] committed:
Merge branch 'develop' into develop--test_bugs-pytest

mergify[bot] committed:
Merge branch 'develop' into develop--test_bugs-pytest

mergify[bot] committed:
Merge branch 'develop' into develop--test_bugs-pytest
